src/single-spa-vue.js

```diff
@@ -47,6 +51,10 @@ function mount(opts, mountedInstances, props) {
         otherOptions.el = props.domElement;
       }
 
+      if (!opts.appOptions.render && !opts.appOptions.template && opts.rootComponent) {
+        otherOptions.render = (h) => h(opts.rootComponent)
+      }
+
```

Comment:
In order for this condition to be satisfied you have to have the following configuration:
```js
const vueLifecycles = singleSpaVue({
  Vue,
  loadRootComponent: () => import(/* webpackChunkName: "planets-app" */'./App.vue').then(property('default')),
  appOptions: {
    el: '#planets',
    router: configuredRouter,
  }
})
```